src/interface_r/R/auto_generated_wrappers.R

```diff
@@ -219,199 +215,119 @@ h2o4gpu.gradient_boosting_regressor <- function(
-	backend = "auto") {
-
-  model <- h2o4gpu$Lasso(
-    alpha = alpha,
     fit_intercept = fit_intercept,
-    normalize = normalize,
-    precompute = precompute,
-    copy_X = copy_X,
-    max_iter = as.integer(max_iter),
+    lambda_min_ratio = lambda_min_ratio,
+    n_lambdas = as.integer(n_lambdas),
+    n_folds = as.integer(n_folds),
+    n_alphas = as.integer(n_alphas),
     tol = tol,
-    warm_start = warm_start,
-    positive = positive,
-    random_state = as_nullable_integer(random_state),
-    selection = selection,
-    n_gpus = as.integer(n_gpus),
+    tol_seek_factor = tol_seek_factor,
+    lambda_stop_early = lambda_stop_early,
     glm_stop_early = glm_stop_early,
     glm_stop_early_error_fraction = glm_stop_early_error_fraction,
-    verbose = verbose,
-    backend = backend
-  )
-  h2o4gpu_model(model, c("regressor"))
-}
-
-#' @export
-h2o4gpu.ridge_regressor <- function(
-	alpha = 1.0,
-	fit_intercept = TRUE,
-	normalize = FALSE,
-	copy_X = TRUE,
-	max_iter = 5000L,
-	tol = 0.01,
-	solver = "auto",
-	random_state = NULL,
-	n_gpus = -1L,
-	glm_stop_early = TRUE,
-	glm_stop_early_error_fraction = 1.0,
-	verbose = FALSE,
-	backend = "auto") {
-
-  model <- h2o4gpu$Ridge(
-    alpha = alpha,
-    fit_intercept = fit_intercept,
-    normalize = normalize,
-    copy_X = copy_X,
     max_iter = as.integer(max_iter),
-    tol = tol,
-    solver = solver,
-    random_state = as_nullable_integer(random_state),
-    n_gpus = as.integer(n_gpus),
-    glm_stop_early = glm_stop_early,
-    glm_stop_early_error_fraction = glm_stop_early_error_fraction,
-    verbose = verbose,
-    backend = backend
+    verbose = as.integer(verbose),
+    family = "elasticnet",
+    store_full_path = as.integer(store_full_path),
+    lambda_max = lambda_max,
+    alpha_max = alpha_max,
+    alpha_min = alpha_min,
+    alphas = alphas,
+    lambdas = lambdas,
+    double_precision = double_precision,
+    order = order
   )
   h2o4gpu_model(model, c("regressor"))
 }
 
 #' @export
-h2o4gpu.elastic_net_regressor <- function(
-	alpha = 1.0,
-	l1_ratio = 0.5,
+h2o4gpu.elastic_net_classifier <- function(
+	n_threads = NULL,
+	gpu_id = 0L,
+	n_gpus = -1L,
 	fit_intercept = TRUE,
-	normalize = FALSE,
-	precompute = FALSE,
-	max_iter = 5000L,
-	copy_X = TRUE,
+	lambda_min_ratio = 1e-07,
+	n_lambdas = 100L,
+	n_folds = 5L,
+	n_alphas = 5L,
 	tol = 0.01,
-	warm_start = FALSE,
-	positive = FALSE,
-	random_state = NULL,
-	selection = "cyclic",
-	n_gpus = -1L,
+	tol_seek_factor = 0.1,
 	lambda_stop_early = TRUE,
 	glm_stop_early = TRUE,
 	glm_stop_early_error_fraction = 1.0,
-	verbose = FALSE,
-	backend = "auto") {
+	max_iter = 5000L,
+	verbose = 0L,
+	 store_full_path = 0L,
+	lambda_max = NULL,
+	alpha_max = 1.0,
+	alpha_min = 0.0,
+	alphas = NULL,
+	lambdas = NULL,
+	double_precision = NULL,
+	order = NULL) {
 
-  model <- h2o4gpu$ElasticNet(
-    alpha = alpha,
-    l1_ratio = l1_ratio,
+  model <- h2o4gpu$ElasticNetH2O(
+    n_threads = n_threads,
```

Comment:
@terrytangyuan This should be `as_nullable_integer(n_threads)` so we need to change something in the `gen_wrappers.R` function.  Seems like we could change [this line](https://github.com/h2oai/h2o4gpu/blob/master/src/interface_r/scripts/gen_wrappers.R#L19):

```
common_nullabl_int_params <- c("random_state")
```
to 
```
common_nullabl_int_params <- c("random_state", "n_threads")
```
By the way, what does `n_threads = NULL` mean?  Does it mean that we will use all threads available on the machine?  In H2O, we have the default `nthreads = -1` which means use nthreads == n cores on the machine.

Reply:
@terrytangyuan @navdeep-G On the other hand, should we even have an `n_threads` parameter in ElasticNet?  I guess that's only available on ElasticNet, but not the other algos?  Do you know if XGBoost will use multiple threads by default?  The other algos do not have `n_threads`, they just have `n_gpus`.  

And I guess if want to keep `n_threads` here, since the other algos don't have it, then it might be a problem to add it to `common_nullabl_int_params` in `gen_wrappers.R` and we'll have to explicitly make this a nullable int.